### PR TITLE
[Bgu Fix]LPS-178297 When we have multiple Java based action executors in the Workflow Extensions project, we cannot get the desired one correctly

### PR DESCRIPTION
**Pull request by W2jx**
Description: https://liferay.slack.com/archives/D04SX8A7EPR/p1678445476081579
issue: https://issues.liferay.com/browse/LPS-178297

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:help

**Comment by W2jx** [comment]
ci:test:workflow

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/action/executor/ActionExecutorManagerImpl.java [hunk 1]
```diff
@@ -97,7 +110,14 @@ private String _getActionExecutorKey(KaleoAction kaleoAction) {
 			return actionType.name();
 		}
 
-		return kaleoAction.getScriptLanguage();
+		String scriptLanguage = kaleoAction.getScriptLanguage();
+
+		if (Objects.equals(ScriptLanguage.JAVA.getValue(), scriptLanguage)) {
+			return scriptLanguage + StringPool.COLON +
+				StringUtil.trim(kaleoAction.getScript());
```
Comment (dantewang) [inline modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/action/executor/ActionExecutorManagerImpl.java:117]: @W2jx Is `kaleoAction.getScript()` always the class name of the action executor?

Comment (W2jx) [reply modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/action/executor/ActionExecutorManagerImpl.java:117]: Hi @dantewang , I used the original logic.

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
@tinatian This logic about script type "java" was removed when we originally made the change to convert method reference to service tracker collection in this class. However, this is actually needed as explained in the ticket, so this pull adds the original logic back.

**Comment by tinatian** [comment]
@W2jx Please add test to use "java" language to make sure the new added fix is working

**Pull request closed without merging**